consensus-peer/consensus/consensus.proto

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright IBM Corp. 2016 All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+		 http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+
+package consensus;
+
+import "google/protobuf/empty.proto";
+
+service atomic_broadcast {
+    rpc broadcast(message) returns (google.protobuf.Empty) {}
```

Comment:
under what conditions would you reject requests?  I guess for the rpc version, this could be useful.  However, the rcp version is significantly slower than the streaming version.